packages/medusa/src/strategies/batch-jobs/price-list/import.ts

```diff
@@ -45,20 +47,30 @@ class PriceListImportStrategy extends AbstractBatchJobStrategy {
   protected readonly priceListService_: PriceListService
   protected readonly batchJobService_: BatchJobService
   protected readonly productVariantService_: ProductVariantService
+  protected readonly featureFlagRouter_: FlagRouter
 
   protected readonly csvParser_: CsvParser<
     PriceListImportCsvSchema,
     Record<string, string>,
     Record<string, string>
   >
 
+  protected get remoteQuery(): RemoteQueryFunction {
+    return this.__container__.remoteQuery
+  }
+
+  protected get pricingModuleService(): IPricingModuleService {
+    return this.__container__.pricingModuleService
+  }
```

Comment:
todo: Since the medusa app loader is assigning a default value for the remote link and I think for all modules, is that necessary? and if yes then we should change where the default value is done and remove this